packages/expo/src/DataMigrationHelper.ts

```diff
@@ -0,0 +1,93 @@
+import Constants from 'expo-constants';
+import * as FileSystem from 'expo-file-system';
+import { Platform } from 'react-native';
+
+export interface ConflictResolver {
+    onConflict(legacyFile: string, currentFile: string): Promise<void>;
+};
```

Comment:
Why are you proposing this resolver as and interface with one property?
More js-friendly style is to go with straight function declaration, isn't it?
see this: https://www.typescriptlang.org/docs/handbook/interfaces.html#function-types
```ts
export interface OnConflictCallback {
  (legacyFile: string, currentFile: string): Promise<void>;
}
```